tests/utils/browser.js

```diff
@@ -32,7 +35,121 @@ const getCookies = (...cookieNameList) => {
   return browser.getCookies(cookieNameList);
 };
 
+const createDoc = async (doc) => {
+  const { err, result } = await browser.executeAsync((doc, callback) => {
+    const db = window.CHTCore.DB.get();
+    return db
+      .put(doc)
+      .then(result => callback({ result }))
+      .catch(err => callback({ err }));
+  }, doc);
+
+  if (err) {
+    throw err;
+  }
+
+  return result;
+};
```

Comment:
Can we come up with a way to do this without the boilerplate? Ideally we'd have a private utility function which means this function can be simplified down to...

```js
const createDoc = async (doc) => {
  return await fancyWrapperWithGoodName((doc) => {
    const db = window.CHTCore.DB.get();
    return db.put(doc);
  }, doc);
});
```

Needs a little varargs work, but would simplify each of these functions so you can actually see what they're doing.